src/views/Today.tsx

```diff
@@ -1,7 +1,26 @@
-import type { Component } from "solid-js";
+import { Component, createComputed, createSignal } from 'solid-js';
 
-const Today: Component = () => {
-  return <div class="">TODAY</div>;
+import type { DayType } from '../components/day-card-list/types';
+
+// Components
+import { DayCardList } from '../components';
+
+interface TodayProps {
+  forecast: DayType | unknown;
+}
+
+const Today: Component<TodayProps> = props => {
+  const [list, setList] = createSignal<any>([]);
```

Comment:
interesting feature createComputed.

maybe it is better to replace lines 13 - 17 with
`  const list = createMemo(() => (props.forecast ? [props.forecast] : []));
`

Reply:
Why not. It should work as well. I believe. Again, tried to use as much of API as possible.